generator/src/html/script/Main.hx

```diff
@@ -18,8 +18,26 @@ class Main {
 		// parse and locate myselft
 		assert(tocData != null, "toc data is missing");
 		var toc = J(JQuery.parseHTML(tocData));
-		assert(document.URL.startsWith(document.baseURI));
-		var myUrl = document.URL.replace(document.baseURI, "").replace(window.location.hash, "");
+		var baseURI = document.baseURI;
+
+		//FIXME IE workaround for the lack of document.baseURI
+		if (baseURI == null) {
+			if (document.getElementsByTagName("base").length > 0){
+				var splitdocpath = document.URL.split("/");
+				var docpath = splitdocpath.join("/");
+				var base = document.getElementsByTagName("base")[0].attributes.getNamedItem("href").value;
+				if (base.startsWith(".")){
+					baseURI = haxe.io.Path.normalize(docpath + "/" + base) + "/" ;
+					if (document.URL.startsWith("file:///"))
```

Comment:
Note: this handles `file:///...`, but not LAN mounts (e.g. `\\\VBOXSRV\...`).

I accidentally stumbled on this limitation because I'm using shared directories (via a virtual LAN) to quickly iterate develop/test with a Windows VM.

Still, no need to fix this here.  Let's check if the solution planned in #116 doesn't already fix this as well.